### PR TITLE
Docs 11153 - Partner Manager v2.7

### PR DESCRIPTION
**Pull request by dhirsch1**

### CODE REVIEW

File: modules/ROOT/pages/configure-host.adoc [hunk 1]
```diff
@@ -10,7 +10,19 @@ Use the host page in Partner Manager to:
 * Create and manage source and target xref:partner-manager-create-message-type.adoc[message types]
 * Configure details associated with the xref:setup-payload-storage-API.adoc[Payload Storage API]
 
+== EDIFACT Global Receive Settings
```
Comment (vijayang) [inline modules/ROOT/pages/configure-host.adoc:13]: The "EDIFACT Global Receive Settings" should be a separate sub-topic under Configure Host.

File: modules/ROOT/pages/configure-partner.adoc [hunk 2]
```diff
@@ -30,13 +30,9 @@ Before you create your first trading partner, ensure you configure your xref:set
 The *New partner* window appears.
 . In the *New partner* window:
 .. In the *Name* field, enter a name that easily identifies your partner.
-.. In the *Type* list, select one of the following identifiers:
-* *AS2 Identity*
-* *Reference ID*
-* *X12-GS*
-* *X12-ISA*
+.. In the *Type* list, select an identifier.
```
Comment (vijayang) [inline modules/ROOT/pages/configure-partner.adoc]: Please change as below:
In the *Type* list, select an identifier type.

File: modules/ROOT/pages/create-outbound-message-flow.adoc [hunk 3]
```diff
@@ -67,32 +68,32 @@ The translation map is added to the message flow.
...
-... From *Message type*, select the message type.
-... From schema select *Use standard X12-<version>-<message-type>* or select *Customize* to define a custom schema.
-... Click *Save*. +
+* To use an existing message type, select the message type from the list and click *Select*.
+* To create a new message type, click *New* next to the search box, define the xref:partner-manager-create-message-type.adoc#create-message-type[EDIFACT and X12 message type], and click *Save*.
++
 The message type and its validation settings are added to the message flow.
-. In the *Acknowledgment* section, select *Expect functional acknowledgment* if you expect your partner to return functional acknowledgments (997).
-.. In the *Select an endpoint: Receive acknowledgment from <parter>* window, click *Select* to specify the endpoint from which to receive the acknowledgment transactions from your partner. +
+.  If your partner expects your organization to return functional acknowledgements:
```
Comment (vijayang) [inline modules/ROOT/pages/create-outbound-message-flow.adoc]: This line needs to be removed

File: modules/ROOT/pages/create-outbound-message-flow.adoc [hunk 4]
```diff
@@ -67,32 +68,32 @@ The translation map is added to the message flow.
...
+* xref:endpoint-as2-receive.adoc[AS2]
+* xref:endpoint-ftp-receive.adoc[FTP]
+* xref:endpoint-sftp-receive.adoc[SFTP]
 ... Click *Save*.
-... In *Mark overdue after*, accept the default value of *24 Hours* or configure values based on when you want the outbound transaction to be designated as overdue for an acknowledgment, according to the SLAs you have with your trading partner. +
+.. In the *Mark overdue after* field, accept the default value of *24 Hours* or configure values based on when you want the outbound transaction to be designated as overdue for an acknowledgment, according to the SLAs you have with your trading partner.
++
 When a functional acknowledgment is not received within the specified timer value, Partner Manager marks the outbound transactions as `Overdue`.
-. In the *X12 Settings* drop-down list, select the X12 settings.
+. In the *<protocol> Settings* list, select the settings.
```
Comment (vijayang) [inline modules/ROOT/pages/create-outbound-message-flow.adoc]: Please change protocol Settings to message format settings

File: modules/ROOT/pages/edi-ack-reconciliation.adoc [hunk 5]
```diff
@@ -40,12 +40,12 @@ Filter for transmissions based on their acknowledgment status:
 +
 image::partner-manager-review-ack.png[Expand the transmission and click Review to enter notes about action taken and mark the transmission as reviewed]
 +
-When you are reviewing a 997 rejection, you see the X12 997 payload received from your trading partner.
+When you review an X12 997 or CONTRL rejection, you see the X12 997 or EDIFACT CONTRL payload received from your trading partner. For example (for X12 997):
```
Comment (vijayang) [inline modules/ROOT/pages/edi-ack-reconciliation.adoc]: Please change as: (EDIFACT in front of CONTRL)
When you review an X12 997 or EDIFACT CONTRL rejection, you see the X12 997 or EDIFACT CONTRL payload received from your trading partner. For example (for X12 997):

File: modules/ROOT/pages/create-map-inbound-edifact-json-xml.adoc [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+. Launch Anypoint Studio.
+. From the *File* menu, select *New > Project from Template*.
+. Log in to Anypoint Exchange with your Anypoint Platform credentials.
+. Search for `edifact`.
+. Click the template named `Template to create inbound and outbound EDIFACT message transformation`.
+. Do either of the following:
+* Click *Open*, to create a new project in your Studio workspace that is preloaded with the template.
+* Download the template from Exchange and open the Mule configuration file `01-inbound-edifact-transform` in the downloaded template.
+
+== Set the Source JSON or XML Message Type
```
Comment (vijayang) [inline modules/ROOT/pages/create-map-inbound-edifact-json-xml.adoc]: This should be change to "Set the Source EDIFACT Message Type"

File: modules/ROOT/pages/create-map-inbound-edifact-json-xml.adoc [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
++
+image
++
+. Select *File* and save the DataWeave map with a meaningful name, such as `INB-EDIFACT-D97A-ORDERS-to-Canonical-Purchase-Order-JSON.dwl`:
++
+image
++
+. Download a https://anypoint.mulesoft.com/exchange/works.integration/b2b-edifact-transform-demo/[sample] project from Exchange that has pre-built mapping to transform D97A ORDERS messages into a JSON application message structure.
+////
+Where do you find this?
```
Comment (vijayang) [inline modules/ROOT/pages/create-map-inbound-edifact-json-xml.adoc]: Within the downloaded example project, the inbound EDIFACT to JSON mapping is available in the Mule configuration 01-inbound-edifact-json, in the Transform component. The dataweave file is saved in src/main/resources/maps/EDIFACT_D97A_ORDERS_to_Canonical_PurchaseOrder_JSON.dwl

File: modules/ROOT/pages/create-map-json-xml-to-outbound-edifact.adoc [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+To create the mapping:
+
+. Click the edit current target icon on top of the right side pane:
++
+image
++
+Select *File*, and save the DataWeave map with a meaningful name, such as: `OTB-Canonical-Purchase-Order-Ack-JSON-to-EDIFACT-D97A-ORDRSP.dwl`:
++
+image
+
```
Comment (vijayang) [inline modules/ROOT/pages/create-map-json-xml-to-outbound-edifact.adoc:66]: We need to add a bullet below referencing the example project (similar to how we are doing in the inbound map topic).

Download a https://anypoint.mulesoft.com/exchange/works.integration/b2b-edifact-transform-demo/[sample] project from Exchange that has pre-built mapping to transform canonical JSON message structure into D97A ORDRSP messages.
Within the downloaded example project, the outbound JSON to EDIFACTmapping is available in the Mule configuration 02-outbound-json-edifact, in the Transform component. The dataweave file is saved in src/main/resources/maps/Canonical_PO_Ack_JSON_to_EDIFACT_D97A_ORDRSP

File: modules/ROOT/pages/edifact-receive-read-settings.adoc [hunk 9]
```diff
@@ -0,0 +1,147 @@
...
+|Character encoding
+a|Indicates the character encoding for messages. This character encoding is used for both send and receive messages.
+
+Possible values include:
+
+* `ASCII`
+* `ISO8859_1`
+* `UTF-8`
+|Disabled
+|===
```
Comment (vijayang) [inline modules/ROOT/pages/edifact-receive-read-settings.adoc]: This topic needs to be replaced with the content in: https://docs.google.com/document/d/16J6CubEMk1mAnr5JWOtRFMjIshCA6xnQthYg3Je4yMA/edit#

Comment (vijayang) [inline modules/ROOT/pages/edifact-receive-read-settings.adoc]: @dhirsch1 - This topic needs to be replaced with the content from https://docs.google.com/document/d/18s4ohdBeGkHkwDS1EDWAEyEfHFsNFbMFExnw-RXVKcY/edit

File: modules/ROOT/pages/edifact-send-settings.adoc [hunk 10]
```diff
@@ -0,0 +1,170 @@
...
+* *CRLF* +
+Carriage Return & Line Feed
+* *LF* +
+Line Feed
+|`CRLF`
+|===
+
+== See Also
+
+* xref:create-outbound-message-flow.adoc[Create and Configure an Outbound Message Flow]
```
Comment (vijayang) [inline modules/ROOT/pages/edifact-send-settings.adoc]: @dhirsch1 - this topic needs to be replaced with content from https://docs.google.com/document/d/16J6CubEMk1mAnr5JWOtRFMjIshCA6xnQthYg3Je4yMA/edit#

**Review by vijayang — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/create-map-inbound-edifact-json-xml.adoc [hunk 11]
```diff
@@ -0,0 +1,83 @@
+= Create a Map For Inbound EDIFACT to JSON or XML
+
+Use the MuleSoft template named `Template to create inbound and outbound EDIFACT message transformation` to help you map EDI EDIFACT transactions to an application message format, such as JSON or XML.
+
+This template uses the DataWeave to create the mapping between the inbound EDI EDIFACT transactions and the application format. It uses the application JSON or XML schema as input for the mapping.
```
Comment (valkyrie69) [inline modules/ROOT/pages/create-map-inbound-edifact-json-xml.adoc]:
```suggestion
This template uses DataWeave to create the mapping between the inbound EDI EDIFACT transactions and the application format. It uses the application JSON or XML schema as input for the mapping.
```

File: modules/ROOT/pages/edi-ack-reconciliation.adoc [hunk 12]
```diff
@@ -1,10 +1,10 @@
 = EDI Acknowledgment Reconciliation
 
-As your trading partners acknowledge your messages, Partner Manager automatically and updates their acknowledgment  status. If you want to be notified, you must specifically configure acknowledgment settings when you create the message.
+As your trading partners acknowledge your messages, Partner Manager automatically and updates their acknowledgment status. If you want to be notified, you must specifically configure acknowledgment settings when you create the message.
```
Comment (valkyrie69) [inline modules/ROOT/pages/edi-ack-reconciliation.adoc]:
```suggestion
As your trading partners acknowledge your messages, Partner Manager automatically updates their acknowledgment status. If you want to be notified, you must specifically configure acknowledgment settings when you create the message.
```

File: modules/ROOT/pages/edifact-send-settings.adoc [hunk 13]
```diff
@@ -0,0 +1,110 @@
...
+|If enabled, adds a UNA segment at the beginning of the generated EDIFACT payload
+|Enabled
+|Enforce valid characters
+|Enforces valid characters
+|Enabled
+|Enforce length limits
+| Validates data elements against length limits defined in the used EDI schema
+|Enabled
+|Expect CONTRL for each transmission sent to <partner>
+|When enabled, sets UNB-09 to `1`, indicating that a technical acknowledgement is expected in the partner's response
```
Comment (valkyrie69) [inline modules/ROOT/pages/edifact-send-settings.adoc]:
```suggestion
|When enabled, sets UNB-09 to `1`, indicating that a technical acknowledgment is expected in the partner's response
```

**Review by valkyrie69 — APPROVED** [review]

**Review by vijayang — APPROVED** [review]

**Pull request merged**